### PR TITLE
Keyvault lab fix

### PR DESCRIPTION
**Pull request by unaihuete93**
Modified lab instructions with UI updates

### CODE REVIEW

File: labs/vstsextend/azurekeyvault/readme.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ In this lab, you will see how you can use Azure Key Vault in a pipeline.
 1. Refer the [Getting Started](../Setup/) page before you begin following the exercises.
 
 1. Use the [Azure DevOps Demo Generator](https://azuredevopsdemogenerator.azurewebsites.net/?name=keyvault){:target="_blank"} to provision the project on your Azure DevOps organization.
-   This URL will automatically select **Azure Key Vault** template in the demo generator. If you want to try other projects, use this URL instead -[azuredevops generator](https://azuredevopsdemogenerator.azurewebsites.net/)
+   This **URL will automatically select Azure Key Vault** template in the demo generator. If you want to try other projects, use this URL instead -[azuredevops generator](https://azuredevopsdemogenerator.azurewebsites.net/)
+
+   ![](images/template-kv.png)
```
Comment (sriramdasbalaji) [inline labs/vstsextend/azurekeyvault/readme.md:28]: Hi, Looks like you forgot to upload some images. Please check

Comment (unaihuete93) [reply labs/vstsextend/azurekeyvault/readme.md:28]: yes sorry, may have forgotten that specific image :( just uploaded it.

Comment (sriramdasbalaji) [reply labs/vstsextend/azurekeyvault/readme.md:28]: Hi, Could you please reverify the images? Still, some images are missing kv-create.png , select-sp.png . 

Thanks

Comment (unaihuete93) [reply labs/vstsextend/azurekeyvault/readme.md:28]: oh sorry, I though it was only that one. I will verify all of them ;)

Comment (unaihuete93) [reply labs/vstsextend/azurekeyvault/readme.md:28]: images added on the last commit

**Review by sriramdasbalaji — APPROVED** [review]

**Pull request merged**